quickwit/quickwit-config/src/node_config/mod.rs

```diff
@@ -439,6 +439,10 @@ impl IngestApiConfig {
 
     fn validate(&self) -> anyhow::Result<()> {
         self.replication_factor()?;
+        ensure!(
+            self.shard_throughput_limit <= PIPELINE_THROUGHPUT,
+            "shard_throughput_limit must be less than {PIPELINE_THROUGHPUT}"
+        );
```

Comment:
This check is promised in this comment: https://github.com/quickwit-oss/quickwit/blob/6ec11c555aaf05912e1c7a394ad6ace48d095696/quickwit/quickwit-control-plane/src/indexing_scheduler/mod.rs#L146-L147